tests/integration/components/partner/client-info-plan/component-test.js

```diff
@@ -0,0 +1,110 @@
+import {
+  module,
+  test
+} from 'qunit';
+import {
+  setupRenderingTest
+} from 'ember-qunit';
+import {
+  render
+} from '@ember/test-helpers';
+import {
+  hbs
+} from 'ember-cli-htmlbars';
+import {
+  setupMirage
+} from "ember-cli-mirage/test-support";
+import {
+  setupIntl
+} from 'ember-intl/test-support';
+import dayjs from 'dayjs';
+import faker from 'faker';
+
+module('Integration | Component | partner/client-info-plan', function (hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+  setupIntl(hooks);
+
+  hooks.beforeEach(async function () {
+    await this.server.createList('organization', 2);
+    await this.owner.lookup('service:organization').load();
+    // await this.owner.lookup('service:partner').load();
+  });
+
+  test('it renders', async function (assert) {
+    await render(hbs `<Partner::ClientInfoPlan/>`);
+    assert.dom('div[data-test-key]').hasText(`t:paymentPlan:()`);
+  })
+
+  test('Render per app plan with projects count and expires on', async function (assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    const clientPlan = this.server.create('partnerclient-plan', {
+      limitedScans: false,
+      expiryDate: dayjs(faker.date.future()).toISOString()
+    });
+    console.log('clientPlan', clientPlan)
+    this.set('clientPlan', clientPlan)
+    await render(hbs `<Partner::ClientInfoPlan @clientPlan={{this.clientPlan}}/>`);
+
+    assert.dom('div[data-test-plan-type]').hasText(`t:perApp:()`);
+    assert.dom('div[data-test-plan-type]').hasStyle({
+      color: 'rgb(89, 149, 239)',
+      'background-color': 'rgb(242, 247, 255)',
+      'border-color': 'rgb(237, 244, 255)'
+    })
```

Comment:
Completely agree!
thought of adding this test case is like, there is two different color options for each plan type. 
Ideally, this could be solved by validating the applied class (dynamically applied) but didn't like of adding an additional attribute for this alone. 
It could be solved by an other option like, adding value for data test attr with active plan type!

Reply:
Not a clean solution though but we could use regexp: `assert.dom('[data-test-plan-type]').hasClass(/.*per-app*/);`

Reply:
@guganabu can you check ember-css-modules tests, they have a way of importing style module.

Reply:
@Praseetha-KR Yes we do have option to import styles inside the script via
https://github.com/salsify/ember-css-modules#programmatic-styles-access
Updated the same with this PR.